perfkitbenchmarker/package_managers.py

```diff
@@ -109,7 +116,18 @@ def Startup(self):
 
   def InstallEpelRepo(self):
     """Installs the Extra Packages for Enterprise Linux repository."""
-    self.InstallPackages('epel-release')
+    try:
+      self.InstallPackages('epel-release')
+    except errors.VmUtil.SshConnectionError as e:
+      stdout, _ = self.RemoteCommand('cat /etc/redhat-release')
```

Comment:
What's the format for `/etc/redhat-release`? Any way to make this regex more specific?